src/index.html

```diff
@@ -11,7 +11,151 @@
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+            Full date of birth:
+          <input type="date" name="age">
+        </label>
+        <label class="form-field">
+            I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input type="password" name="password">
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="pet"
+              value="yes"
+            > Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="pet"
+              value="no"
+            > No
+          </label>
+        </div>
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            id="favcolor"
+            name="favcolor"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            id="time"
+            name="time"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            id="cartype"
+            name="cartype"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Tesla">Tesla</option>
+            <option
+              value="Choose"
+              disabled
+              selected
+            >
+            </option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+            min="0"
+            max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
```

Comment:
```suggestion
      </fieldset>
      
      <fieldset class="group">
```
It would be great to add empty lines between blocks of code to make your code easier to read